src/wooting-rgb-sdk.c

```diff
@@ -49,7 +53,8 @@ static uint8_t gammaFilter[256] = {
     215, 218, 220, 223, 225, 228, 231, 233, 236, 239, 241, 244, 247, 249, 252,
     255};
 
-static uint16_t rgb_buffer_matrix[WOOTING_RGB_ROWS][WOOTING_RGB_COLS] = {0};
+static WOOTING_RGB_MATRIX rgb_buffer_matrix_array[WOOTING_MAX_RGB_DEVICES];
+static WOOTING_RGB_MATRIX *rgb_buffer_matrix;
 static bool rgb_buffer_matrix_changed = false;
```

Comment:
This flag feels pretty redundant now if you're not keeping track of it on a per-matrix basis. It's not hard to see how you can break this:
e.g. 
1. Change some colours
2. Change device
3. Send colours (it gets sent even when the colours for that device haven't changed)
4. Change device back
5. Send colours (it doesn't get sent due to `rgb_buffer_matrix_changed` being false, when it actually does have changes)

This flag needs to be either per buffer or removed entirely (thus putting the burden of not sending frames unnecessarily up to the SDK user)

Reply:
It's removed entirely. I personally thing it should not be up to the SDK to keep track of this. It should just enable controlling the RGB of Wooting keyboards.